Formula/u/unitycatalog.rb

```diff
@@ -0,0 +1,45 @@
+class Unitycatalog < Formula
+  desc "Open, Multi-modal Catalog for Data & AI"
+  homepage "https://unitycatalog.io/"
+  url "https://github.com/unitycatalog/unitycatalog/archive/refs/tags/v0.3.0.tar.gz"
+  sha256 "fae708a22f1e38e19f754aca22925d66016a7efeaab680ce87c27496c75078d1"
+  license "Apache-2.0"
+
+  depends_on "openjdk@17"
+
+  def install
+    system "build/sbt", "createTarball"
+
+    mkdir "build" do
+      system "tar", "xzf", "../target/unitycatalog-#{version}.tar.gz", "-C", "."
+
+      inreplace "jars/classpath" do |s|
+        s.gsub! %r{[^:]+/([^/]+\.jar)}, "#{libexec}/jars/\\1"
+      end
+
+      bin.install Dir["bin/*"]
+      libexec.install "jars"
+      (etc/"unitycatalog").install "etc"
+    end
+
+    env = Language::Java.java_home_env("17")
+    env["PATH"] = "${JAVA_HOME}/bin:${PATH}" if OS.linux?
+    bin.env_script_all_files libexec/"bin", env
```

Comment:
this should be good enough?

```suggestion
    bin.env_script_all_files libexec/"bin", Language::Java.overridable_java_home_env("17")
```

Reply:
Changed to `overridable_java_home_env`, but CI fails on Linux with `java_home_env` without `PATH`, so I'd expect the same to happen.